crates/matrix-sdk/src/sliding_sync.rs

```diff
@@ -464,7 +467,11 @@ impl<'a> SlidingSyncViewRequestGenerator<'a> {
     ) -> Raw<sliding_sync_events::SyncRequestList> {
         let sort = Some(self.view.sort.clone());
         let required_state = Some(self.view.required_state.clone());
-        let timeline_limit = self.view.timeline_limit.clone().map(|v| v.try_into().expect("u32 always fits into UInt"));
+        let timeline_limit = self
+            .view
+            .timeline_limit
+            .clone()
+            .map(|v| v.try_into().expect("u32 always fits into UInt"));
```

Comment:
If `v` is really `u32` then `.try_into().expect("…")` can be replaced by `.into()`.